src/templates/tsconfig.module.json

```diff
@@ -5,7 +5,7 @@
     "declarationMap": true,
     "emitDeclarationOnly": true,
     "esModuleInterop": true,
-    "lib": ["ES2022"],
+    "lib": ["ES2022", "DOM"],
```

Comment:
I thought we did not want the dom type definitions? 

https://www.typescriptlang.org/tsconfig/#lib

```
You may want to change these for a few reasons:

Your program doesn’t run in a browser, so you don’t want the "dom" type definitions
```

Reply:
Yeah, gotta think a bit on this one. I noticed that in Pepr's `.tsconfig.json` file we added `DOM` but not in the templates. That seems reasonable, but not including `DOM` causes issues with running a `pepr build` once `markdownlint-cli` was added as a dependency due to a typing issue with `katex` (a transitive).

Why that causes the build of a new module to [fail](https://github.com/defenseunicorns/pepr/actions/runs/13249499731/job/36983848870) in the first place seems weird, since they don't share devDeps.

Reply:
Got it, im not sure that this is bad either. I saw that this one time had "DOM" included.

I vaguely believe I remember that this _may_ have messed up intellisense on the module.. We have to check bc I do not remember anymore

Reply:
I finally found the convo. I did this to allow intellisense around WASM files in October of 2023. Jeff made a point that this explodes the amount of types for the end user. 

https://github.com/defenseunicorns/pepr/pull/318#issuecomment-1764611965

Reply:
Looks like we were able to do this without including DOM after all!